modules/fontrendering/include/modules/fontrendering/util/fontutils.h

```diff
@@ -57,12 +55,42 @@ getAvailableFonts(const std::string& fontPath = std::string());
  *
  * @return default path containing fonts shipped with Inviwo. Corresponds to the font directory
  *                 located in the font rendering module directory.
- *
  */
 std::string IVW_MODULE_FONTRENDERING_API getDefaultFontPath();
 
+/**
+ * \brief returns the default typeface of Inviwo along with the default font path and extension if
+ * \p fullPath is true
+ *
+ * @param fullPath    the function returns the full path of the default font if true
+ * @return default font name
+ */
+std::string IVW_MODULE_FONTRENDERING_API getDefaultFont(bool fullPath = false);
```

Comment:
Nice! But I avoid using a book it reads really badly at the call sites. Either add two functions or use an enum, could also use a enum for the different kind of fonts, bold, caption, etc.

Reply:
Also all four functions should have the path option I think...